docs/tutorials/05_animation/viz_keyframe_bezier_interpolator.py

```diff
@@ -0,0 +1,192 @@
+                           order_transparent=True)
+showm.initialize()
+
+###############################################################################
+# Note: If a control point is set to `None`, it gets the value of the
+# point it controls.
+keyframes = [
+    # time - position - pre control point  - post control point
+    [0.0, [-2, 0, 0], None, [-15, 6, 0]],
+    [5.0, [18, 0, 0], [27, 18, 0], [27, -18, 0]],
+    [9.0, [-5, -10, -10], None, None]
+]
+
+
+###############################################################################
+# Initializing the timeline
+timeline = Timeline(playback_panel=True)
+sphere = actor.sphere(np.array([[0, 0, 0]]), (1, 0, 1))
+timeline.add_actor(sphere)
+
+###############################################################################
+# Setting Cubic Bezier keyframes
+
+for keyframe in keyframes:
+
+    ###########################################################################
+    # visualizing the points and control points (only for demonstration)
+    t = keyframe[0]
+    pos = keyframe[1]
+    pre_control_point = keyframe[2]
+    post_control_point = keyframe[3]
+
+    ###########################################################################
+    # setting position keyframe
+    timeline.set_position(t, pos, in_cp=pre_control_point, out_cp=post_control_point)
+
+    ###########################################################################
+    # visualizing position keyframe
+    vis_point = actor.sphere(np.array([pos]), (1, 0, 0), radii=0.3)
+    scene.add(vis_point)
+
+    ###########################################################################
+    # Visualizing the control points and their length (if exist)
+    for cp in [pre_control_point, post_control_point]:
+        if cp is not None:
+            vis_cps = actor.sphere(np.array([cp]), (0, 0, 1),
+                                   radii=0.6)
+            cline_actor = actor.line(np.array([[pos, cp]]),
+                                     colors=np.array([0, 1, 0]))
+            scene.add(vis_cps, cline_actor)
+
+###############################################################################
+# changing position interpolation into cubic bezier interpolation
+timeline.set_position_interpolator(cubic_bezier_interpolator)
+
+###############################################################################
+# adding actors to the scene
+scene.add(timeline)
+
+
+###############################################################################
+# making a function to update the animation
+def timer_callback(_obj, _event):
+    timeline.update_animation()
+    showm.render()
+
+
+###############################################################################
+# Adding the callback function that updates the animation
+showm.add_timer_callback(True, 10, timer_callback)
+
+if interactive:
+    showm.start()
+
+window.record(scene, out_path='viz_keyframe_animation_bezier_2.png',
+              size=(900, 768))
```

Comment:
The first animation is working.

The second animation generate a `Segmentation fault: 11`